src/ui/ActiveScripts/ActiveScriptsRoot.tsx

```diff
@@ -3,29 +3,78 @@
-import Tabs from "@mui/material/Tabs";
-import Tab from "@mui/material/Tab";
+import { Button, Tabs, Tab } from "@mui/material";
 
 import { ActiveScriptsPage } from "./ActiveScriptsPage";
 import { RecentScriptsPage } from "./RecentScriptsPage";
+import { RecentErrorsPage } from "../../ErrorHandling/RecentErrorsPage";
 import { useRerender } from "../React/hooks";
+import { errorModalsAreSuppressed, ErrorState, toggleSuppressErrorModals } from "../../ErrorHandling/ErrorState";
+import { OptionSwitch } from "../React/OptionSwitch";
+import { killAllScripts } from "../../Netscript/killWorkerScript";
+import { SimplePage } from "@enums";
+import { Router } from "../GameRoot";
 
-export function ActiveScriptsRoot(): React.ReactElement {
-  const [tab, setTab] = useState<"active" | "recent">("active");
+type ActiveScriptsTab = SimplePage.ActiveScripts | SimplePage.RecentlyKilledScripts | SimplePage.RecentErrors;
+
+export type ComponentProps = {
+  page: ActiveScriptsTab;
+};
+
+export function ActiveScriptsRoot({ page }: ComponentProps): React.ReactElement {
+  const [tab, setTab] = useState<ActiveScriptsTab>(ErrorState.UnreadErrors > 0 ? SimplePage.RecentErrors : page);
   useRerender(400);
 
-  function handleChange(event: React.SyntheticEvent, tab: "active" | "recent"): void {
+  function handleChange(
+    __event: React.SyntheticEvent,
+    tab: SimplePage.ActiveScripts | SimplePage.RecentlyKilledScripts | SimplePage.RecentErrors,
+  ): void {
     setTab(tab);
+    Router.toPage(tab);
   }
+
+  function errorTabText(): string {
+    if (!ErrorState.UnreadErrors || tab === SimplePage.RecentErrors) {
+      return "Recent Errors";
+    }
+    return `Recent Errors (${ErrorState.UnreadErrors})`;
+  }
+
   return (
     <>
-      <Tabs variant="fullWidth" value={tab} onChange={handleChange} sx={{ minWidth: "fit-content", maxWidth: "25%" }}>
-        <Tab label={"Active"} value={"active"} />
-        <Tab label={"Recently Killed"} value={"recent"} />
-      </Tabs>
+      <div style={{ display: "flex", alignItems: "center" }}>
+        <Tabs
+          value={tab}
+          onChange={handleChange}
+          sx={{
+            minHeight: "fit-content",
+            "& .MuiButtonBase-root.MuiTab-root": {
+              margin: 0,
+              padding: "10px",
+              whiteSpace: "pre",
+              minHeight: "40px",
+            },
+          }}
+        >
+          <Tab label={"Active"} value={SimplePage.ActiveScripts} />
+          <Tab label={"Recently Killed"} value={SimplePage.RecentlyKilledScripts} />
+          <Tab label={errorTabText()} value={SimplePage.RecentErrors} />
+        </Tabs>
+        <OptionSwitch
+          checked={errorModalsAreSuppressed()}
+          onChange={(newValue) => toggleSuppressErrorModals(newValue)}
+          text="Prevent error modals"
+          tooltip={<>If this is set, no error modals will be shown until the game is reloaded.</>}
+          wrapperStyles={{ marginLeft: "20px" }}
+        />
```

Comment:
```suggestion
        <OptionSwitch
          checked={errorModalsAreSuppressed()}
          onChange={(newValue) => toggleSuppressErrorModals(newValue)}
          text="Suppress error modals (5 minutes)"
          tooltip={
            <>
              If this is set, no error modals will be shown for the next five minutes.
              <br />
              The "Suppress error modals" toggle in the game options will always suppress modals, and only log errors to
              the Recent Errors page.
            </>
          }
          wrapperStyles={{ marginLeft: "20px" }}
        />
```
I reused the text in `src\ErrorHandling\ErrorModal.tsx`. Note that I use `"Suppress error modals (5 minutes)"` instead of `"Suppress error modals for 5 minutes"` to reduce the length of the switch a bit.